content/en/synthetics/uptime_check.md

```diff
@@ -141,6 +141,24 @@ A check is considered `FAILED` if it doesn't satisfy the assertions configured f
 | UNABLE_TO_GET_ISSUER_CERT_LOCALLY  | The issuer certificate of a locally found certificate is not found. This usually means that the list of trusted certificates is not complete.                         |
 | UNABLE_TO_VERIFY_LEAF_SIGNATURE    | No signatures are verified because the certificate chain contains only one certificate, which is not self-signed, and the issuer is not trusted.                      |
 
+## External metrics
+
+The following metrics are generated by uptime checks.
+
+| | |
+| --- | --- |
+| **synthetics.http.response.time** <br/> (gauge) | Overall time needed for the request to be processed. This is an independent metric; it is not computed as a sum of all the different network timing metrics (DNS, SSL, download, etc.) A warning appears in check runner logs if this value is too different from the sum of all network timing metrics. <br/> _shown as milliseconds_ |
```

Comment:
> A warning appears in check runner logs if this value is too different from the sum of all network timing metrics.
This is purely internal, let's remove that sentence :) 

> This is an independent metric; it is not computed as a sum of all the different network timing metrics (DNS, SSL, download, etc.) 
Unsure if we want that to be specified externally, I would remove it for now and only leave `Overall time needed for the request to be processed`.